src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -32,42 +32,42 @@
   'skip_probability_crop' : 0.1,
   'min_object_covered': 0.0,
   'min_eject_coverage': 0.5,
-  'resize_method': 'turicreate',
```

Comment:
Why are we switching this back? Is this just a merge issue? (We need to resolve the merge conflicts with master)